pkg/gofr/context.go

```diff
@@ -75,6 +84,40 @@ func (c *Context) WriteMessageToSocket(data any) error {
 	return conn.WriteMessage(websocket.TextMessage, message)
 }
 
+func (c *Context) GetAuthInfo() AuthInfo {
+	return c
+}
+
+func (c *Context) GetClaims() jwt.MapClaims {
+	claims, ok := c.Request.Context().Value(middleware.JWTClaim).(jwt.MapClaims)
+	if !ok {
+		c.Warn("failed to get claims from context")
+		return nil
+	}
+
+	return claims
+}
+
+func (c *Context) GetUsername() string {
+	claims, ok := c.Request.Context().Value(middleware.Username).(string)
+	if !ok {
+		c.Warn("failed to get username from context")
```

Comment:
I wouldn't use the failed wording on a non-blocking error.

You are using a warning error level, so I would use something like


```suggestion
		c.Warn("no username found in context")
```

Reply:
But I'm unsure about the logic of logging here

For me, there should be no log, it should be either:
- returning an error when missing
- returning simply a void result, like you do

In both cases, it should be from caller responsibility to handle this case

Imagine this log entry could be repeated fof ever and ever